addons/binding/org.openhab.binding.mysensors/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,34 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MySensors Binding
+Bundle-SymbolicName: org.openhab.binding.mysensors;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
```

Comment:
Everywhere else we are referencing 1.8. Should the RequiredExecutionEnvironment be JavaSE-1.8 too?